docs/intro.rst

```diff
@@ -63,35 +69,58 @@ The ``mbdump-derived.tar.bz2`` archive contains annotations, user tags and searc
 These archives include all the data required for setting up an instance of
 CritiqueBrainz.
 
-You can automatically download the archives (``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``) and
-begin the import for the MusicBrainz database::
+There are two ways in which you can import the database dump:-
 
-   $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+1. By downloading and importing the database dump simultaneously::
 
-.. note::
+    $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+
+2. Or by manually downloading the dumps and then import it:-
+
+i. For this, you have to download the dumps ``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``
+   from https://musicbrainz.org/doc/MusicBrainz_Database/Download.
```

Comment:
I'd prefer a direct link to the files instead of naming files and providing a single link